internal/app/go-filecoin/internal/submodule/chain_submodule.go

```diff
@@ -64,7 +64,7 @@ func NewChainSubmodule(ctx context.Context, config chainConfig, repo chainRepo,
 	}
 	actorState := consensus.NewActorStateStore(chainStore, blockstore.CborStore, blockstore.Blockstore, processor)
 	messageStore := chain.NewMessageStore(blockstore.Blockstore)
-	chainState := cst.NewChainStateReadWriter(chainStore, messageStore, blockstore.CborStore, builtin.DefaultActors)
+	chainState := cst.NewChainStateReadWriter(chainStore, messageStore, blockstore.Blockstore, builtin.DefaultActors)
```

Comment:
We accepts a blockstore here since it allows the state tree to be easily exported using a dag service, which requires a blockstore.